server/routerlicious/package-lock.json

```diff
@@ -1,8 +1,14019 @@
 {
   "name": "root",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
You should use Node 12.x which is still on lockFileVersion 1 (and this is the version we say we support, so this will help prevent you from introducing any dependency on futuristic Node features)